File: app/src/main/java/com/nimtego/itunes/data/entity/Album.java
package com.nimtego.itunes.data.entity;

import lombok.Builder;
import lombok.Data;


@Data
@Builder
public class Album {
    private Integer albumId;
    private Integer albumArtistId;
    private String albumName;
    private String albumRealiseDate;
    private Integer albumTrackCount;
    private String albumArtWorkUrl;
    private String albumArtistName;
    private Double albumPrice;
}

File: app/src/main/java/com/nimtego/plectrum/presentation/mvp/model/song/Album.kt
package com.nimtego.plectrum.presentation.mvp.model.song

data class Album(
    val albumId: Int,
    val albumArtistId: Int,
    val albumName: String,
    val albumRealiseDate: String,
    val albumTrackCount: Int,
    val albumArtWorkUrl: String,
    val albumArtistName: String,
    val albumPrice: Double)